StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetLoader.swift

```diff
@@ -118,9 +118,7 @@ final class PaymentSheetLoader {
             }
         }
 
-        if linkAccountService.hasSessionCookie {
-            return try await lookUpConsumerSession(email: nil)
-        } else if let email = linkAccountService.getLastSignUpEmail() {
+        if let email = linkAccountService.getLastSignUpEmail() {
```

Comment:
I'm probably overthinking this, but do we need to be concerned with any cases where lastSignUpEmail's value is different than the email on the defaultBillingDetails.email.  The code is clear in that the lastSignUpEmail is prioritized, but I'm guessing that "could" fail technically, and we may want to retry with the defaultBillingDetails.email (if it happens to be a different value) ?

Reply:
Possibly, yeah. Not sure if I want to keep making network requests in that case, as these delay PaymentSheet's load and the failure case here is showing a generic Link button.

Though I *just* found out that we're about to change this behavior again! I'll draft this for now.